pkg/operation/seed/seed.go

```diff
@@ -168,3 +165,47 @@ func (s *Seed) GetValidVolumeSize(size string) string {
 
 	return size
 }
+
+// GetLoadBalancerServiceAnnotations returns the load balancer annotations set for the seed if any.
+func (s *Seed) GetLoadBalancerServiceAnnotations() map[string]string {
+	seed := s.GetInfo()
+	if seed.Spec.Settings != nil && seed.Spec.Settings.LoadBalancerServices != nil {
+		return seed.Spec.Settings.LoadBalancerServices.Annotations
+	}
+	return map[string]string{}
+}
+
+// GetLoadBalancerServiceExternalTrafficPolicy indicates the external traffic policy for the seed if any.
+func (s *Seed) GetLoadBalancerServiceExternalTrafficPolicy() *corev1.ServiceExternalTrafficPolicyType {
+	seed := s.GetInfo()
+	if seed.Spec.Settings != nil && seed.Spec.Settings.LoadBalancerServices != nil {
+		return seed.Spec.Settings.LoadBalancerServices.ExternalTrafficPolicy
+	}
+	return nil
+}
+
+// GetZonalLoadBalancerServiceAnnotations returns the zonal load balancer annotations set for the seed if any.
+func (s *Seed) GetZonalLoadBalancerServiceAnnotations(zone string) map[string]string {
+	seed := s.GetInfo()
+	if seed.Spec.Settings != nil && seed.Spec.Settings.LoadBalancerServices != nil {
+		for _, zoneSettings := range seed.Spec.Settings.LoadBalancerServices.ZoneSettings {
+			if zoneSettings.ZoneName == zone {
+				return zoneSettings.Annotations
+			}
+		}
+	}
+	return map[string]string{}
+}
+
+// GetZonalLoadBalancerServiceExternalTrafficPolicy indicates the zonal external traffic policy for the seed if any.
+func (s *Seed) GetZonalLoadBalancerServiceExternalTrafficPolicy(zone string) *corev1.ServiceExternalTrafficPolicyType {
+	seed := s.GetInfo()
+	if seed.Spec.Settings != nil && seed.Spec.Settings.LoadBalancerServices != nil {
+		for _, zoneSettings := range seed.Spec.Settings.LoadBalancerServices.ZoneSettings {
+			if zoneSettings.ZoneName == zone {
+				return zoneSettings.ExternalTrafficPolicy
+			}
+		}
+	}
+	return nil
```

Comment:
same question here

Reply:
Makes even more sense for the external traffic policy. Updated accordingly in the next commit.